srcs/juloo.keyboard2/Keyboard2View.java

```diff
@@ -285,25 +376,38 @@ private int labelColor(KeyValue k, boolean isKeyDown, int defaultColor)
           return _theme.activatedColor;
       }
     }
+    else if (isSubKeyDown)
+    {
+      return _theme.activatedColor;
+    }
     return defaultColor;
   }
 
-  private void drawLabel(Canvas canvas, KeyValue k, float x, float y, float keyH, boolean isKeyDown)
+  private void drawLabel(Canvas canvas, KeyValue k, float x, float y, float keyH, boolean isSubKeyDown, boolean isKeyDown)
   {
     float textSize = scaleTextSize(k, _config.labelTextSize, keyH);
     k = KeyModifier.handleFlags(k, _flags);
     Paint p = _theme.labelPaint(((k.flags & KeyValue.FLAG_KEY_FONT) != 0));
-    p.setColor(labelColor(k, isKeyDown, _theme.labelColor));
+    p.setColor(labelColor(k, isSubKeyDown, isKeyDown, _theme.labelColor));
     p.setTextSize(textSize);
     canvas.drawText(k.symbol, x, (keyH - p.ascent() - p.descent()) / 2f + y, p);
   }
 
-  private void drawSubLabel(Canvas canvas, KeyValue k, float x, float y, float keyW, float keyH, Paint.Align a, Vertical v, boolean isKeyDown)
+  private void drawSubLabel(Canvas canvas, KeyValue k, float x, float y, float keyW, float keyH, Paint.Align a, Vertical v, boolean isSubKeyDown, boolean isKeyDown)
   {
+    if (_config.accent_fn && (_flags & KeyValue.FLAG_FN) == 0)
```

Comment:
This just hides the symbols but the keys are still here. I've done something similar in a57bdf8 but it needs more work (hitting a removed key will trigger a vibration and won't hit the middle symbol).

Reply:
I planned to replace the keys but I chose to do it like this as I didn't have much time and it was easier to implement. I still use some of the accents sometimes even though they can't be seen and use fn to remind myself if I've forgotten where one is.

Reply:
I've improved the removing of keys to be able to implement F11 and F12. See e872c58 and https://github.com/Julow/Unexpected-Keyboard/commit/ba6d62dee2a1c0dae63eb758551b089a7cd9ffc1